lldb/unittests/Host/SocketTest.cpp

```diff
@@ -85,6 +85,43 @@ TEST_P(SocketTest, DomainListenConnectAccept) {
   std::unique_ptr<DomainSocket> socket_b_up;
   CreateDomainConnectedSockets(Path, &socket_a_up, &socket_b_up);
 }
+
+TEST_P(SocketTest, DomainMainLoopAccept) {
```

Comment:
Add `#if LLDB_ENABLE_POSIX`. Otherwise it will fail on Windows.

Reply:
It's already in a ifdef block, next to the other domain socket test. That's why you don't see an ifdef in the default view.